tests/integration/test_fast_color_refine.py

```diff
@@ -4,17 +4,17 @@
 import os
 import unittest
 
-from color_refinement import fast_color_refine
```

Comment:
Dit is nodig, want anders is

https://github.com/MinThaMie/graph_isomorphism/pull/53/files#diff-2d4fa821df8b8741c332fe820221b934R65

onmogelijk. Stomme Python! 🐍